### ISSUE TITLE
Improve WPF Tooltips Hover for Accessibility 

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1014]
Copied from ADO ID: 917236

Repro Steps:

1. Launch affected app referenced in VSO ID
2. Navigate to various actionable UI in the toolbar.
3. Check if the tooltip displayed is satisfying the below conditions:​
         Hoverable​
         Persistent

-affected app has accessibility updates enabled in config file.
-affected app just uses ToolTip property on button or RibbonButton to add tooltip.  Changing app to use tooltip object via <Button.Tooltip><Tooltip/></Button.Tooltip> does not affect behavior.

Expected Result:

Hoverable: Should be able to hover on the additional content(tooltip)​
Persistent: The additional content(tooltip) should be visible until the hover or focus trigger is removed, the user dismisses it, or its information is no longer valid

Actual Result:

The tooltip is not Hoverable. Tooltip is dismissed on hovering to it.​  
The tooltip is not persistent. Tooltip is dismissed after some time without removing the focus or hover.

Initial Discussion from team research:
I don't believe we have any logic in place to detect hover on the tooltip and keep it alive, when the mouse moves out of the tooltip's parent element the tooltip is automatically dismissed.

**Comment by arpitmathur** [comment]
This feature request is currently out of 3.0. We should evaluate this issue for 3.1 or 5.0.

**Comment by jlukosky** [comment]
Shouldn't this be tagged _Accessibility MAS_ as two of the three requirements of MAS 1.4.13: Content on Hover or Focus? Removing the label may certainly be appropriate but I'm not sure of the reasoning.

**Comment by SamBent** [comment]
Repurposing this issue to track the WCAG 2.1 ToolTip work (which subsumes the work for which this issue was opened).  See the [spec](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/specs/tooltip.md).

**Comment by ryalanms** [comment]
This is fixed by @SamBent's change.
